cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,121 @@
+<div class="license-wrapper">
+    <h3 class="label setting-label">
+        <%= gettext("License Type") %>
+    </h3>
+    <ul class="license-types">
+        <% var link_start_tpl = '<a href="{url}" target="_blank">'; %>
+        <% _.each(licenseInfo, function(license, licenseType) { %>
+            <li data-license="<%- licenseType %>">
+                <button name="license-<%- licenseType %>"
+                        class="action license-button <% if(model.type === licenseType) { print("is-selected"); } %>"
+                        <% if (license.tooltip) { %>data-tooltip="<%- license.tooltip %>"<% } %>>
```

Comment:
@singingwolfboy the current implementation of data-tooltip at edX is inaccessible to keyboard and screen reader users.  I recommend against using it until we can fix it.  I haven't taken a look at how its implemented in the platform, so I don't know how difficult it will be to do uniformly.  The preferred method is to have have on-screen text for the tip (for sighted keyboard users) and using ```aria-describedby="[IDref of the tip]"``` to programatically associate that text with the button control (for screen reader users).  Alternatively, if you didn't want it displayed on screen at all, you could put the tip offscreen (class="sr") (still requires aria-describedby) and make the tooltip display when the button has :focus as well as on :hover.  Since this is a problem across the platform, you can either punt or apply one of the solutions I outlined above.